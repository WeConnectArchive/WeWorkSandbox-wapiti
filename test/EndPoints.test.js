import React from 'react';
import {EndPoints} from '../build/wapiti.module';
import renderer from 'react-test-renderer';

describe('EndPoints', () => {

  const thePath = [{"test1":"test1"},{"test2":"test2"}]

  const component = renderer.create(
     <EndPoints
        selectedServer  = ""
        paths           = {thePath}
        specType        = "rest"
     ></EndPoints>
  );

  let tree;

  it('renders with rest', () => {
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
