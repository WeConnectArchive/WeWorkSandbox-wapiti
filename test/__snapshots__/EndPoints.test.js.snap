// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EndPoints renders with rest 1`] = `
Array [
  <span />,
  <div
    className="Endpoint_m-endpoint__2nB-9 FontStyles_regular-font__3G8fz"
  >
    <div
      className="Endpoint_head__DuPOZ"
      onClick={[Function]}
    >
      <div
        className="Endpoint_method__10Wmy"
      >
         
         
      </div>
      <div
        className="Endpoint_path__2RJZJ"
      >
         
         
      </div>
      <div
        className="Endpoint_only-large-screen__18TAV"
        style={
          Object {
            "flex": "1",
            "minWidth": "60px",
          }
        }
      />
      <div
        className="Endpoint_m-markdown-small__9vYUT Endpoint_descr__3BSCz"
      >
         
        
      </div>
    </div>
  </div>,
  <span />,
  <div
    className="Endpoint_m-endpoint__2nB-9 FontStyles_regular-font__3G8fz"
  >
    <div
      className="Endpoint_head__DuPOZ"
      onClick={[Function]}
    >
      <div
        className="Endpoint_method__10Wmy"
      >
         
         
      </div>
      <div
        className="Endpoint_path__2RJZJ"
      >
         
         
      </div>
      <div
        className="Endpoint_only-large-screen__18TAV"
        style={
          Object {
            "flex": "1",
            "minWidth": "60px",
          }
        }
      />
      <div
        className="Endpoint_m-markdown-small__9vYUT Endpoint_descr__3BSCz"
      >
         
        
      </div>
    </div>
  </div>,
]
`;
